enhancements/20220525-system-logging.md

```diff
@@ -0,0 +1,174 @@
+# System Logging
+
+## Summary
+
+We need to be able to support exporting harvester system logs outside the cluster, to allow for collecting and analyzing those logs to aid in debugging and harvester management.
+
+### Related Issues
+
+- https://github.com/harvester/harvester/issues/577
+- https://github.com/harvester/harvester/issues/2644
+- https://github.com/harvester/harvester/issues/2645
+- https://github.com/harvester/harvester/issues/2646
+- https://github.com/harvester/harvester/issues/2647
+
+## Motivation
+
+### Goals
+
+List the specific goals of the enhancement. How will we know that this has succeeded?
+
+- [X] The user can aggregate harvester logs in a centralized location
+  - [X] k8s cluster logs
+  - [X] host system logs (ie `/var/log`)
+- [X] The user can export the aggregated harvester logs outside the cluster (ex rancher)
+
+### Non-goals
+
+- this enhancement does not cover integration into the harvester / rancher UI (but ideally this will eventually be implemented as well)
+
+## Proposal
+
+We will deploy a new `ManagedChart` in the [Harvester Installer](https://github.com/harvester/harvester-installer) to
+install. The `ManagedChart` will deploy a `ClusterFlow` to select and aggregate interesting logs, which can be managed
```

Comment:
please update:

The `managedchart` will deploy a few PODs to collect logging by default, and add CRD of `ClusterFlow`,`ClusterOutput`, but it will not create any instances, user can create them in Harvester Dashboard.  (Considering the `loki` is removed in day1)